src/status_im/subs/utils.cljs

```diff
@@ -0,0 +1,7 @@
+(ns status-im.subs.utils
```

Comment:
let's avoid very generic namespaces like this :)

This will eventually be filled with completely unrelated functionality. 

At the very least `status-im.subs.chat.utils`